run.go

```diff
@@ -328,17 +328,17 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 		return err
 	}
 
+	ep := b.Entrypoint()
```

Comment:
spell out endpoint.  I read three lines down and forgot what ep stood for.

Reply:
Ok it actually looks like the existing code uses ep. Could you change all references to ep to be endpoint.

Reply:
Yep, just used existing code. Updated in 757d1e00dbcd66aa523c8e7b7a0fb345f207d043.